src/foam/comics/DAOController.js

```diff
@@ -266,7 +266,7 @@ foam.CLASS({
           result = 'data:text/csv;charset=utf-8,' + result;
           var encodedUri = encodeURI(result);
           var link = document.createElement('a');
-          link.setAttribute('href', encodedUri);
+          link.setAttribute('href', encodedUri.replace(/#/g, '%23'));
```

Comment:
why not fix this in encodeURI?

Reply:
encodeURI is a build-in function. I can fix this on line 267 with 
var encodedUri = encodeURI(result).replace(/#/g, '%23')